### ISSUE TITLE
RepositoryFileApi.updateFile V3 expected response code should be OK

### ISSUE THREAD

**Issue by richardmward** [gmessner/gitlab4j-api#82]
I believe the V3 method should be expecting `Response.Status.OK`

```
public RepositoryFile updateFile(RepositoryFile file, Integer projectId, String branchName, String commitMessage) throws GitLabApiException {
        Form formData = file2form(file, branchName, commitMessage);
        Response response = isApiVersion(ApiVersion.V3) ?
            put(Response.Status.CREATED, formData.asMap(), "projects", projectId, "repository", "files") :
            put(Response.Status.CREATED, formData.asMap(), "projects", projectId, "repository", "files", urlEncode(file.getFilePath()));
        return (response.readEntity(RepositoryFile.class));
}
```

**Comment by gmessner** [comment]
@richardmward 
This has been fixed and tested in the just-released version 4.6.1

**Comment by richardmward** [comment]
I think the fix to this has broken the `createFile` method, which does expect `Response.Status.CREATED` when using the `V3` API.

**Comment by gmessner** [comment]
This will be fixed in the next day or so and unit tests will be added to validate.

**Comment by gmessner** [comment]
createFile() has been fixed for the V3 API and is available in release 4.6.2
